stablehlo/reference/Interpreter.cpp

```diff
@@ -138,6 +138,13 @@ llvm::Expected<SmallVector<Tensor>> eval(func::FuncOp func,
       for (Value ssaOperand : returnOp.getOperands())
         runtimeOperands.push_back(fetchOperand(ssaOperand));
       return runtimeOperands;
+    } else if (auto selectOp = dyn_cast<SelectOp>(op)) {
```

Comment:
Can you rebase this on top of the new PR with support for the regions? The changes in this file will need to be migrated to Ops.cpp.

Reply:
Ah thanks! Rebased and updated.
As I see, the conflict is not hard to address, so I will not rebase other PRs and resolve them when submit.